library/L2_HAL/sensors/battery/test/ltc4150_test.cpp

```diff
@@ -1,38 +1,60 @@
 #include "L2_HAL/sensors/battery/ltc4150.hpp"
 #include "L4_Testing/testing_frameworks.hpp"
+#include "L1_Peripheral/interrupt.hpp"
+#include "L1_Peripheral/gpio.hpp"
+#include "utility/units.hpp"
 
 namespace sjsu
 {
 #define APPROX_EQUALITY(expected, actual, resolution) \
   CHECK(-resolution < (actual - expected));           \
   CHECK((actual - expected) < resolution);
```

Comment:
Rather than using this macro, you can use the Catch2's `Approx()` matcher instead.

Reply:
All the uses of `Approx()` work great, except for the last test case. I scheduled a meeting with you today to show the problem I am encountering,

Reply:
Done! Thanks for showing me [epsilon](https://github.com/catchorg/Catch2/blob/master/docs/assertions.md#floating-point-comparisons)!!